doc/MissingFeatures.md

```diff
@@ -0,0 +1,43 @@
+## Missing Features
+
+The following is a list of features missing to match the support from [oneAPI oneMKL BLAS](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onemkl/source/domains/blas/blas).
+The order does not imply any priority.
+
+For questions regarding input types or operators support, please refer to this [documentation](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onemkl/source/domains/blas/blas)
```

Comment:
Maybe, it is possible to remove the duplicate link and compact everything in a single paragraph.
 
```suggestion
The following is a list of missing features for supporting entirely the oneAPI oneMKL BLAS interface. Their order does not reflect any kind of priority. Please, refer to this [link](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onemkl/source/domains/blas/blas) for details regarding input types or operators support.
```

Reply:
I agree on the possibility of removing the double link, since it is the same, but I like having `the order does not imply any priority` standing out a little bit more. Maybe we can rephrase it as:
```suggestion
The following is a list of features missing in portBLAS for supporting entirely the [oneAPI oneMKL BLAS interface](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onemkl/source/domains/blas/blas).
The order does not reflect any kind of priority.

For questions regarding input types or operators support, please refer to this the link above.
```
How do you see it in this way?

Reply:
Sure, fine.